### PR TITLE
Make button translate-able like the other button

### LINKED ISSUE #2266: Button is not translate-able in setup proces
In the 2nd step of the setup proces, the "Create pages" button is unable to be translated.

### PR DESCRIPTION
**Pull request by NekoJonez**
Make the "Create selected pages" button translate-able.

Fixes #2266 

### Changes proposed in this Pull Request

Makes a button translate-able.

### CODE REVIEW

**Comment by tripflex** [comment]
Thanks for the PR @NekoJonez but you're missing the closing `"` please update the PR and include the closing quote

File: includes/admin/views/html-admin-setup-step-2.php [hunk 1]
```diff
@@ -68,7 +68,7 @@
 		<tfoot>
 		<tr>
 			<th colspan="4">
-				<input type="submit" class="button button-primary" value="Create selected pages" />
+				<input type="submit" class="button button-primary" value="<?php esc_html_e( 'Create selected pages', 'wp-job-manager' ); ?> />
```
Comment (tripflex) [inline includes/admin/views/html-admin-setup-step-2.php]: You're missing the closing `"`

**Comment by NekoJonez** [comment]
> Thanks for the PR @NekoJonez but you're missing the closing `"` please update the PR and include the closing quote

Should be done :) @tripflex

**Comment by tripflex** [comment]
Thanks looks good to me 👍

**Review by tripflex — APPROVED** [review]

File: includes/admin/views/html-admin-setup-step-2.php (hunk 1 above)
Comment (NekoJonez) [reply includes/admin/views/html-admin-setup-step-2.php]: It should be added now.

**Pull request merged**